napari/components/dims.py

```diff
@@ -426,9 +440,14 @@ def _focus_down(self):
     def _roll(self):
```

Comment:
I don't mind the new `rollable` state in `Dims` too much, but if that wasn't well received by the wider community, we could instead store that state in the axis model and pass it through here. That might be tricky though, as the lifetime of the axis model is probably bound to the dialog, which closes fairly often.

So don't do anything yet, but wanted to make a note of this idea in case it's useful in the future.